### DISCUSSION TITLE
Account blocked

### DISCUSSION THREAD

**Discussion by Th3Wall** [vercel/community#218] [Help] [answered]
Hi guys,
I'm writing you because I've just found out that my personal account has been blocked without any particular reason and without any notice or informative email.
I can't understand why this is happening, I've been using Vercel a lot and I've always supported it since the beginning. I don't think I have violated the rules since I have always used it for uploading just the demos of my websites or side-projects.

Could you please help me understand why my account has been blocked and please help me in order to recover it?
I have some websites deployed that I need to put back online as soon as possible.

I am hoping for a positive response from you.
Have a nice day!

**Comment by RizBizKits** [comment, marked as answer]
Hey, are you able to get to send us an email at registration@vercel.com (with identifiable info about your account e.g username) so the team can take a look at why this was blocked? Thanks!

**Reply by Th3Wall** [reply]
Hey @RizBizKits, thanks for replying!
Sure, I'm sending an email to that address in minutes. Hope to solve this problem.

Thanks,
Th3Wall
